src/components/Calculator.module.css

```diff
@@ -0,0 +1,43 @@
+@supports (display: grid) {
+  .calculator {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr 1fr;
+  }
+}
+
+/* In case the user is on IE */
+@supports (display: -ms-grid) {
+  .calculator {
+    display: -ms-grid;
+    -ms-grid-columns: 1fr 1fr 1fr 1fr;
+    -ms-grid-rows: 1fr 1fr 1fr 1fr;
+  }
+}
+
+div p {
+  padding: 20px;
+  margin: 0;
+  font-size: 1.5rem;
+}
+
+.calculator__input {
+  grid-column: 1 / 5;
+  background-color: #858693;
+  text-align: right;
+  color: white;
+}
+
+.calculator__button__grey {
+  background-color: #e0e0e0;
+  text-align: center;
+}
+
+.calculator__button__orange {
+  background-color: #f39148;
+  text-align: center;
+}
+
+.calculator__button__zero {
+  grid-column: 1 / 3;
+}
```

Comment:
- [ ] Kindly make sure your design match that of the guideline
Your Implementation | Guideline Design
:---:|:---:
![screenshot (29)](https://user-images.githubusercontent.com/44624138/158488337-99edf995-e176-4320-9e4a-373b265d3378.png) | ![screenshot (28)](https://user-images.githubusercontent.com/44624138/158488401-1ab6d109-18dd-45b2-8dc7-0bc6e11fe8c7.png)

Reply:
My reasoning behind was that the calculator may be adaptable to different sizes in a real scenario, should I constrain it's size either way?